tests/sentry/ratelimits/test_cardinality.py

```diff
@@ -8,11 +8,20 @@
     RedisCardinalityLimiter,
     RequestedQuota,
 )
+from sentry.utils import redis
 
 
-@pytest.fixture
-def limiter():
-    return RedisCardinalityLimiter()
+@pytest.fixture(params=["cluster", "rb"])
```

Comment:
I've never seen this automatic parameter-ization with pytest before -- it's nice.